packages/nodejs/src/__tests__/client.test.ts

```diff
@@ -50,7 +62,7 @@ describe("BaseClient", () => {
     expect(startSpy).not.toHaveBeenCalled()
   })
 
-  it("starts the client when the active option is true", () => {
+  it("does not start the client when the active option is true", () => {
```

Comment:
Huh.. I would expect this to be true. This test won't work. It creates a new client, but adds an assertion on the previously created client. I would revert the test name, and make it pending or something. Or at least create an issue for it to fix it.

Reply:
> It creates a new client, but adds an assertion on the previously created client.

Aaah, I missed that! Now it makes sense. I thought it was weird, but I thought, "at least I'll make the name of the test match the behaviour that's under test" -- it turns out the name, was what was wrong!

Will revert this here and fix this test on a separate PR.

Reply:
Done! #613